lib/middleware/redirectDefaultLicence.js

```diff
@@ -0,0 +1,11 @@
+const logger = require('../logger');
+const cookieHandler = require('../helpers/cookieHandler');
+
+module.exports = function redirectDefaultLicence(req, res, next) {
+	if (undefined === req.params.licenceId && req.currentUser && req.currentUser.uuid && req.listOfLicences) {
+		const licenceId = req.listOfLicences[0].licenceId || cookieHandler.getLicenceListCookie(req, res);
```

Comment:
@ciprianlujeru good spot. This is used in the default route where no licence id is provided, is it not?